modules/conversion-center/module.php

```diff
@@ -8,6 +8,7 @@
 use Elementor\Core\Base\Module as BaseModule;
 use Elementor\Core\Documents_Manager;
 use Elementor\Core\Experiments\Manager;
+use Elementor\Core\Wp_Api;
```

Comment:
Why do we need `Wp_Api`?

Reply:
@louiswol94 sorry I had imported it and forgot to delete it